website/docs/user-guides/observability-set-up.md

```diff
@@ -4,4 +4,49 @@ title: Setup Observability with Chaos Center
 sidebar_label: Setup Observability
 ---
 
-## Coming soon
+---
+
+### Overview
+
+Overview Tab helps you to manage your connected data sources and dashboards, all while making it easier to view and analyze your workflows. The overview tab is split into 3 parts -
+
+- Data Sources
+- Application Dashboards
+- Workflow Dashboards
+
+:::note
+When there is no agent connected or in the `active` state, the overview tab will show a banner for connecting an agent.
```

Comment:
This part of connecting an Agent can be moved to `Before you Begin` section and can be linked to Litmusctl connecting and agent page. Also, the image is not required as the UX is very clear here.

Reply:
change language